kinto/core/__init__.py

```diff
@@ -28,6 +28,7 @@
     'backoff': None,
     'batch_max_requests': 25,
     'cache_backend': '',
+    'cache_hosts': '',
```

Comment:
Was this change mandatory to be able to use the KINTO_CACHE_HOSTS env variable? It doesn't work with other backends apparently.